src/lib/styles.js

```diff
@@ -249,6 +250,23 @@ img {
   -webkit-user-select: none;
   -ms-user-select: none;
   user-select: none;
+  width: 100%;
+}
+
+.flex {
```

Comment:
maybe use another class definition, it is not good to see flex to something that is actually set to block